activity_calendar/templates/activity_calendar/slot_blocks/slot_block.html

```diff
@@ -15,7 +15,15 @@ <h5 class="card-title">
                 {% if slot.location %}
                     <div class="row no-gutters">
                         <small class="text-muted">
-                            Alternative Location: {{ slot.location }}
+                            Alternative Location:
+                            {% if not activity_moment.private_slot_locations or user in slot.participants.all %}
+                                {{ slot.location }}
+                                {% if slot.location_is_private %}
```

Comment:
I cannot find any other mention of `location_is_private` in this PR or in the codebase

Reply:
And this is exactly why we have code reviews! It was not moved when the "private"-status of slots was moved to the activity itself.